test/account_objects-test.js

```diff
@@ -39,7 +39,7 @@ function filter_threading_fields(entries) {
 
 /* ---------------------------------- TEST ---------------------------------- */
 
-suite('account_objects', function() {
+suite.skip('account_objects', function() {
```

Comment:
We can't consider this complete with so many tests and suites marked to `skip`. It would be a step backward to disable so many tests.